aws-opentelemetry-distro/src/amazon/opentelemetry/distro/_aws_attribute_keys.py

```diff
@@ -14,6 +14,6 @@
 
 # AWS_#_NAME attributes are not supported in python as they are not part of the Semantic Conventions.
 # TODO：Move to Semantic Conventions when these attributes are added.
-AWS_QUEUE_URL: str = "aws.sqs.queue_url"
-AWS_QUEUE_NAME: str = "aws.sqs.queue_name"
-AWS_STREAM_NAME: str = "aws.kinesis.stream_name"
+AWS_QUEUE_URL: str = "aws.sqs.queue.url"
+AWS_QUEUE_NAME: str = "aws.sqs.queue.name"
+AWS_STREAM_NAME: str = "aws.kinesis.stream.name"
```

Comment:
Not blocking: would be nice if we fixed const names
```suggestion
AWS_SQS_QUEUE_URL: str = "aws.sqs.queue.url"
AWS_SQS_QUEUE_NAME: str = "aws.sqs.queue.name"
AWS_KINESIS_STREAM_NAME: str = "aws.kinesis.stream.name"
```

Reply:
Absolutely.